e2e/_suites/ingest-manager/features/fleet_mode_agent.feature

```diff
@@ -6,13 +6,9 @@ Feature: Fleet Mode Agent
 Scenario: Deploying an agent
   When an agent is deployed to Fleet
   Then the agent is listed in Fleet as online
-    And system package dashboards are listed in Fleet
-
-@start-agent
```

Comment:
I like this removal, because deploying implicitly means starting an agent. Neat!

Reply:
yeah, i still like this too - i think I'll rebase on my branch and push up the remaining changes that still make sense and aren't already implemented (most of it is).  when done we can close this out I think. will post back